rlo/scripts/azureml_util.py

```diff
@@ -29,7 +29,7 @@ def combined_source_directory():
         )
         # Copy ksc in as a subdirectory. Note this does not include ts2ks.
         shutil.copytree(
-            os.path.join(root_dir, "knossos-ksc", "src", "python", "ksc"),
+            os.path.join(os.path.dirname(root_dir), "src", "python", "ksc"),
```

Comment:
Perhaps rename "root_dir" to "rlo_dir"

Reply:
Does `utils.py:get_ksc_dir` help here?

Reply:
Do we even need to copy ksc if rlo is in ksc?